bindata/manifests/daemon/daemonset.yaml

```diff
@@ -21,6 +21,8 @@ spec:
         component: network
         type: infra
         openshift.io/component: network
+      annotations:
+        kubectl.kubernetes.io/default-logs-container: sriov-network-config-daemon
```

Comment:
This one is deprecated: https://kubernetes.io/docs/reference/labels-annotations-taints/#kubectl-kubernetes-io-default-logs-container-deprecated

please use : `kubectl.kubernetes.io/default-container`

Reply:
thanks for catching that. fixed